metron-interface/metron-rest/src/test/java/org/apache/metron/rest/service/impl/AlertsUIServiceImplTest.java

```diff
@@ -126,63 +127,63 @@ public void escalateAlertShouldSendMessageToKafka() throws Exception {
   }
 
   @Test
-  public void getShouldProperlyReturnActiveProfile() throws Exception {
-    when(userSettingsClient.findOne(user1, AlertsUIServiceImpl.ALERT_USER_SETTING_TYPE)).thenReturn(Optional.of(user1AlertUserSettings));
-
-    AlertsUIUserSettings expectedAlertsUIUserSettings = new AlertsUIUserSettings();
-    expectedAlertsUIUserSettings.setTableColumns(Collections.singletonList("user1_field"));
-    assertEquals(expectedAlertsUIUserSettings, alertsUIService.getAlertsUIUserSettings().get());
-    verify(userSettingsClient, times(1)).findOne(user1, AlertsUIServiceImpl.ALERT_USER_SETTING_TYPE);
-    verifyNoMoreInteractions(userSettingsClient);
+  public void shouldGetActiveProfile() throws Exception {
+    AlertsUIUserSettings expected = new AlertsUIUserSettings();
+    expected.setTableColumns(Collections.singletonList("user1_field"));
+
+    // save a profile for current user
+    userSettingsClient.save(user1, ALERT_USER_SETTING_TYPE, toJSON(expected));
```

Comment:
Being able to use a real `HBaseSettingsClient` instead of a mock made these tests much easier to grok and maintain in my opinion.  Previously we had to use a lot of mockito-magic to test this.